modules/Bio/EnsEMBL/Analysis/Hive/Config/HomologyRnaseq.pm

```diff
@@ -0,0 +1,322 @@
+=head1 LICENSE
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::Config::HomologyRnaseq;
+
+use strict;
+use warnings;
+use File::Spec::Functions;
+
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use Bio::EnsEMBL::Analysis::Tools::Utilities qw(get_analysis_settings);
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Analysis::Hive::Config::HiveBaseConfig_conf');
+
+sub default_options {
+  my ($self) = @_;
+  return {
+    # inherit other stuff from the base class
+    %{ $self->SUPER::default_options() },
+
+######################################################
+#
+# Variable settings- You change these!!!
+#
+######################################################
+########################
+# Misc setup info
+########################
+    'dbowner'                         => '' || $ENV{EHIVE_USER} || $ENV{USER},
+    'pipeline_name'                   => '' || $self->o('production_name').'_'.$self->o('ensembl_release'),
+    'user_r'                          => '', # read only db user
+    'user'                            => '', # write db user
+    'password'                        => '', # password for write db user
+    'server_set'                      => '', # What server set to user, e.g. set1
```

Comment:
This can be removed, it should only be used by the main pipeline

Reply:
done